src/span-data.js

```diff
@@ -35,15 +35,19 @@ var uid = 1;
  * @param {number} skipFrames the number of frames to remove from the top of the stack.
  * @constructor
  */
-function SpanData(agent, trace, name, parentSpanId, isRoot, skipFrames) {
+function SpanData(trace, name, parentSpanId, isRoot, skipFrames) {
   var spanId = uid++;
-  this.agent = agent;
   var spanName = traceUtil.truncate(name, constants.TRACE_SERVICE_SPAN_NAME_LIMIT);
   this.span = new TraceSpan(spanName, spanId, parentSpanId);
   this.trace = trace;
   this.isRoot = isRoot;
+  this.serializedTraceContext = traceUtil.generateTraceContext({
+    traceId: this.trace.traceId,
+    spanId: this.span.spanId,
+    options: 1 // always traced; TODO
```

Comment:
Removed TODO -- it was left there earlier to remind me to verify that options should always be set to 1.